fedimint-api/src/lib.rs

```diff
@@ -115,9 +115,12 @@ pub fn sats(amount: u64) -> Amount {
     Amount::from_sats(amount)
 }
 
+/// OutPoint references an output in a transaction uniquely
```

Comment:
maybe ambiguous: it not only references the output uniquely in the transaction context, but globally

Reply:
Also, using a inline code block for the type name would look nicer rendered

Reply:
yeah, I phrased it a bit different now: 

```rs
/// `OutPoint` represents a globally unique output in a transaction
///
/// Hence, a transaction ID and the output index is required. 

```